documentation/UML/sequenceDiagram.puml

```diff
@@ -0,0 +1,37 @@
+@startuml
+
+title "MC Verified MSISDN"
+
+actor "Mobile App\nCustomer" as user
+box "Customer's Device" 
+    participant "Developer's App" as mApp
+end box
+
+box "MC IDGW"
+participant "/authrize" as auth
+participant "/token" as token
+participant "/userinfo" as userInfo
+end box
+
+activate user
+user -> mApp: opens Developer's App
+activate mApp
+mApp -> auth: scope=openid mc_verified_msisdn\n&state={...}\n&{query params defined inside MC specs}
```

Comment:
For me when the scope has been already defined in the standard - Mobile Connect for example - we have to take it as such to avoid confusion for implementation knowing it. I guess this is the case here.